celery/backends/database/models.py

```diff
@@ -30,18 +30,34 @@ class Task(ResultModelBase):
     date_done = sa.Column(sa.DateTime, default=datetime.utcnow,
                           onupdate=datetime.utcnow, nullable=True)
     traceback = sa.Column(sa.Text, nullable=True)
+    name = sa.Column(sa.String(155))
+    args = sa.Column(sa.String(155))
+    kwargs = sa.Column(sa.String(155))
```

Comment:
Sorry for intruding, but I was wondering if there was a specific reason why the size is 155 for these fields?
Especially for the args / kwargs, which can be long.

Reply:
> 
> 
> Sorry for intruding, but I was wondering if there was a specific reason why the size is 155 for these fields?
> Especially for the args / kwargs, which can be long.

No real reason. In my implementation those aren't large fields. If we need them to be larger than 155 characters then we can do that.

Reply:
Make them text fields instead.
The name can be more limited.

Reply:
Changed them to type Text and set them so they could be nullable.